Nuance_MC_Format.txt

```diff
@@ -0,0 +1,74 @@
+NUANCE MC FORMAT
+================
+
+This desribes the format for a file containing "nuance format" kinematic
+information for passing to WCSim.
+
+An example event is shown below:
+$ begin
+$ nuance 3
+$ vertex 811.5 -1030.4 -1681.7 5.86347E+04
+$ track 14 1000.0000 1.00000 0.00000 0.00000 -1
+$ track 2212 935.9840 -0.03219 -0.24434 0.96915 -1
+$ info 2 949000 0.0000E+00
+$ track 13 295.6312 0.15021 0.98821 -0.02950 -2
+$ track 211 427.5731 0.66875 -0.66698 -0.32849 -2
+$ track 2212 1212.7798 0.88658 -0.07320 0.45675 -2
+$ track 13 295.6312 0.15021 0.98821 -0.02950 0
+$ track 2212 1212.7798 0.88658 -0.07320 0.45675 0
+$ track 111 382.1095 -0.73647 -0.35958 -0.57299 0
+$ end
+
+Each line begins with a text tag, indicating the information it contains.
+
+    begin: 
+
+       Marks the start of the event.
+
+    nuance: 
+
+      This line indicates the reaction mode for the event.
+
+    vertex: 
+
+       The first three numbers are the x, y and z coordinate, in cm. The
+    fourth number is the time, in simulation units normally ns.  To specify
+    different units use the command /mygen/time_units in the .mac file.
+
```

Comment:
I think it's good to be specific about how to write files with multiple vertices
```suggestion
Multiple vertices can used in a single event by repeating the vertex, track, and info lines; the nuance line should not be repeated.
```